firmware/examples/PWM/source/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include "L0_LowLevel/delay.hpp"
+#include "L1_Drivers/gpio.hpp"
+#include "L1_Drivers/pwm.hpp"
+#include "L2_Utilities/debug_print.hpp"
+
+int main(void)
+{
+    DEBUG_PRINT("Pwm Application Starting...");
+
+    DEBUG_PRINT("Creating Gpio powered LED...");
```

Comment:
Print out which pin is being powered, so the user running this example doesn't have to look at the code to connect the hardware together.